docs/mathematics.rst

```diff
@@ -0,0 +1,134 @@
+and the probability of that cause to produce that effect.
+Hence, the elements :math:`P(E_{i}|C_{\mu})` represent the probability that a given :math:`C_{\mu}`
+results in the effect :math:`E_i` and thus comprise the known or simulated response matrix :math:`R`.
+The prior :math:`P(C_{\mu})` represents the current or best knowledge of the true (normalized) cause distribution. 
+
+We then can connect the measured observed effects to their causes via
+
+.. math::
+
+    \phi(C_{\mu}) = \sum_{i=1}^{n_{E}} P(C_{\mu} | E_{i}) \ n(E_{i})
+
+
+However, a cause may not necessarily result in any effect, i.e. there is a certain efficiency :math:`\epsilon_{\mu}`
+associated with each cause:
+
+.. math::
+
+    0 \ \leq \ \epsilon_{\mu} \ = \sum_{j=1}^{n_E} \ P(E_j | C_{\mu}) \leq 1
+
+Taking this into account in the estimate of :math:`\phi(C)`, we have
+
+.. math::
+
+    \begin{align}  
+    \phi(C_{\mu}) &= \frac{1}{\epsilon_{\mu}} \sum_{i=1}^{n_{E}} P(C_{\mu} | E_{i}) \ n(E_{i}) \\
+                  &= \sum_{i=1}^{n_{E}} M_{\mu i} \ n(E_{i})
+    \end{align}
+
+
+where the explicit form of :math:`M` is identified as
+
+
+.. math::
+
+
+    M_{\mu j} = \frac{ P(E_{j} | C_{\mu}) \ P(C_{\mu})}{ \left[ \sum_{k}^{n_E} P(E_k | C_{\mu})  \right] \, \left[ \sum_{\nu}^{n_{C}} P(E_{j} | C_{\nu}) \ P(C_{\nu}) \right]}
+
+
+The components of :math:`M_{\mu j}` are:
+
+- :math:`P(E_{j} | C_{\mu})` the known response matrix
+- :math:`n(E_j)` the measured frequency distribution of effects
+- :math:`P(C_{\mu})` the prior distribution of the causes
+
+
+
+--------------------
+Iterative Unfolding
+--------------------
+
+Having obtained the form of :math:`M` and thus an estimate of :math:`\phi(C)`,
+we have a new best guess as to the form of the cause distribution.
+By feeding in :math:`\phi(C)` as an updated prior, we can again get an improved estimate
+of the cause distribution.
+Shortening the notation by keeping Latin subscripts for effect indices and Greek subscripts for the causes, 
+we can then obtain :math:`\phi(C)^{t+1}` at iteration :math:`t+1` via
+
+.. math::
+
+    \begin{align}
+    \phi_{\mu}^{t+1} &= \sum_j M_{\mu j} n_j \\
+    M_{\mu j} &= \frac{P_{\mu j} \, \phi^t_{\mu}}{ \epsilon_{\mu} \sum_{\rho} P_{\rho j} \, \phi^{t}_\rho}
+    \end{align}
+
+
+where :math:`\phi^t` is the cause distribution at iteration :math:`t`, and 
+:math:`P_{\mu j}` are the elements of the **response matrix**.
+Thus we have built an iterative unfolding procedure.
+Using an appropriate test statistic such as a K-S test to compare :math:`\phi^t_{\mu}` 
+and :math:`\phi^{t+1}_{\mu}`, we can decide when subsequent iterations provide negligible improvements,
+and after which the unfolding is said to have converged.
+
+The algorithm which PyUnfold employs is illustrated by the following:
+
+.. image:: _static/unfolding_algorithm.png
+
+
+An exhaustive description and derivation of the propagation of uncertainties can be found in the
+latex documentation provided with PyUnfold.
```

Comment:
Might consider adding a hyperlink to the  latex pdf (pyunfold/docs/latex_writeup/main.pdf)